doc/source/train/user_guide.rst

```diff
@@ -540,26 +540,61 @@ equivalent to the sum of the number of epochs.
     from ray import train
     from ray.train import Trainer
 
+    import torch
+    import torch.nn as nn
+    from torch.optim import Adam
+    import numpy as np
+
+
     def train_func(config):
-        model = 0 # This should be replaced with a real model.
-        for epoch in range(config["num_epochs"]):
-            model += epoch # This should be replaced with training logic.
-            train.save_checkpoint(epoch=epoch, model=model)
+        n = 100
+        # create a toy dataset
+        # data   : X - dim = (n, 4)
+        # target : Y - dim = (n, 1)
+        Y = torch.Tensor(np.random.uniform(0, 1, size=(n, 1)))
+        X = torch.Tensor(np.random.normal(0, 1, size=(n, 4)))
+
+        # toy neural network : 1-layer
+        model = nn.Linear(4, 1)
+        mse = nn.MSELoss()
+        optimizer = Adam(model.parameters(), lr=3e-4)
 
-    trainer = Trainer(backend="torch", num_workers=2)
+        for epoch in range(config["num_epochs"]):
+            # compute loss
+            y = model.forward(X)
+            loss = mse(y, Y)
+            # back-propagate loss
+            optimizer.zero_grad()
+            loss.backward()
+            optimizer.step()
+            # For TensorFlow (Keras) models, use model.get_weights()
```

Comment:
Thanks for leaving this comment here. In the long run we should have individual tabs showing a concrete example for each backend. Could you start by just adding a tab for PyTorch?

```
.. tabs::
  .. group-tab:: PyTorch
```

Reply:
@matthewdeng Thank you for your feedback! I think I can add a TF example. Revising other parts as well.

Reply:
Thanks 💯 for adding the TF example as well!